### PR TITLE
lightbits: build system for lightbits

### PR DESCRIPTION
**Pull request by sashas-lb**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,10 @@
+*.o
+*.ko
+*.o.cmd
+*.ko.cmd
+*tmp_versions
+Module.symvers
+ena.mod.c
+modules.order
+
+
```
Comment (adir-lb) [inline .gitignore:10]: please remove those empty lines.

File: kernel/linux/ena/Makefile [hunk 2]
```diff
@@ -1,3 +1,5 @@
+include $(PWD)/Makefile.env
```
Comment (adir-lb) [inline kernel/linux/ena/Makefile:1]: not sure that using `$(PWD)` is good here. why not inferring the absolute path?

File: kernel/linux/ena/Makefile [hunk 3]
```diff
@@ -12,11 +14,11 @@ ccflags-y := -I$(src)
...
-clean:
-	make -C /lib/modules/$(BUILD_KERNEL)/build M=$(CURDIR) clean
+#BUILD_KERNEL ?= $(shell uname -r)
+#
+#ccflags-y += -Wfatal-errors
+#all:
+#	make -C /lib/modules/$(BUILD_KERNEL)/build M=$(CURDIR) modules
+#
+#clean:
+#	make -C /lib/modules/$(BUILD_KERNEL)/build M=$(CURDIR) clean
```
Comment (adir-lb) [inline kernel/linux/ena/Makefile:24]: why is it in comment?

File: .gitignore (hunk 1 above)
Comment (sashas-lb) [reply .gitignore:10]: done

File: kernel/linux/ena/Makefile (hunk 2 above)
Comment (sashas-lb) [reply kernel/linux/ena/Makefile:1]: all our makefiles are relying that we are running from root dir of project

**Pull request merged**